### PR TITLE
Release v3.0.0 of the Thunderhead SDK for Android

### PR DESCRIPTION
**Pull request by lawrencethompson**
*Updated README
*Added TROUBLESHOOTING-GUIDE
*Updated Artifacts with new version

### CODE REVIEW

File: TROUBLESHOOTING-GUIDE.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            }
+            if(details.requested.group == "com.google.android.gms" && details.requested.name == "play-services-basement") {
+                details.useVersion "15.0.1" // Where 17.0.1 is replaced with the apps required version
+            }
+        }
+    }
+}
+```
+
+## Logcat Errors
```
Comment (andreipop) [inline TROUBLESHOOTING-GUIDE.md]: This section needs to be removed.

File: README.md [hunk 2]
```diff
@@ -899,6 +806,9 @@ aspectj {
 }
 ```
 
+## Integration Guide
```
Comment (andreipop) [inline README.md]: Should say `Troubleshooting Guide`

File: README.md [hunk 3]
```diff
@@ -899,6 +806,9 @@ aspectj {
...
+[Trouble Shooting Guide](TROUBLESHOOTING-GUIDE.md)
```
Comment (andreipop) [inline README.md]: `Troubleshooting Guide`

File: TROUBLESHOOTING-GUIDE.md (hunk 1 above)
Comment (lawrencethompson) [reply TROUBLESHOOTING-GUIDE.md]: Removed

File: README.md (hunk 2 above)
Comment (lawrencethompson) [reply README.md]: Updated

File: README.md (hunk 3 above)
Comment (lawrencethompson) [reply README.md]: Updated

**Comment by lawrencethompson** [comment]
AAR folder has been removed.

**Pull request merged**